src/css/loader.css

```diff
@@ -0,0 +1,47 @@
+:root {
```

Comment:
jak coś potrzebujesz dodać do root'a to zrób to w styles.css
kolory są już tam dodane

Reply:
wiem, ale nie chciałem ingerować w te pliki. Rozumiem że poprawić?